time/src/main/java/io/spine/time/LocalDateTimeTemporal.java

```diff
@@ -26,27 +26,44 @@
 
 package io.spine.time;
 
-import com.google.protobuf.Timestamp;
+import io.spine.annotation.GeneratedMixin;
 
 import java.time.Instant;
 
+import static io.spine.time.LocalDateTimes.converter;
 import static java.time.ZoneOffset.UTC;
 
 /**
  * An implementation of {@link Temporal} based on {@link LocalDateTime}.
- *
- * <p>This interface is designed to be implemented by {@code io.spine.time.LocalDateTime}
- * exclusively. The interface does not add any abstract methods to its message counterpart.
  */
+@GeneratedMixin
 interface LocalDateTimeTemporal extends TemporalMessage<LocalDateTime>, LocalDateTimeOrBuilder {
 
     @Override
-    default Timestamp toTimestamp() {
+    default Instant toInstant() {
         Instant instant = java.time.LocalDateTime
-                .of(LocalDates.toJavaTime(getDate()), LocalTimes.toJavaTime(getTime()))
+                .of(getDate().toJavaTime(), getTime().toJavaTime())
                 .toInstant(UTC);
-        Timestamp result = InstantConverter.instance()
-                                           .convert(instant);
-        return result;
+        return instant;
+    }
+
+    /** Obtains the date part of this date/time instance. */
```

Comment:
I thought we've overall agreed to stop using the shortcut version of javadocs. Event for one-liners.

Reply:
I think we need to be able to write a bit more compact code, and in mixin interfaces in particular. It's just too much of a ceremony to have a 3-line Javadoc for a method the whole purpose of this is to avoid the `get` prefix.